src/main/java/com/softserve/itacademy/controller/AuthController.java

```diff
@@ -0,0 +1,54 @@
+package com.softserve.itacademy.controller;
+
+import com.softserve.itacademy.security.dto.RegistrationDto;
+import com.softserve.itacademy.security.dto.SuccessRegistrationDto;
+import com.softserve.itacademy.security.service.RegistrationService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.core.AuthenticationException;
+import org.springframework.security.web.WebAttributes;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpSession;
+import javax.validation.Valid;
+
+@RestController
+public class AuthController {
+
+    RegistrationService registrationService;
+
+    @Autowired
+    public AuthController(RegistrationService registrationService) {
+        this.registrationService = registrationService;
+    }
+
+    @PostMapping("registration")
```

Comment:
Should it be slash in path "registration"? "/registration"

Reply:
Thx, fixed typo.